sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/GenerateMigrationOutputTask.kt

```diff
@@ -26,21 +25,16 @@ import java.util.ServiceLoader
 
 @CacheableTask
 abstract class GenerateMigrationOutputTask : SqlDelightWorkerTask() {
-  @Suppress("unused")
-  // Required to invalidate the task on version updates.
-  @Input
-  val pluginVersion = VERSION
```

Comment:
Just curious, why did you remove this property? Does Gradle invalides the tasks automatically?

Reply:
I left a message on one of the commits:

> The @classpath annotation on classpath property will invalidate tasks when the files on the classpath change.

Gradle will track the classpath input to monitor for changes, so Gradle will indeed invalidate the tasks automatically if there are changes to the plugin.

There's some more info here: https://docs.gradle.org/8.1.1/userguide/incremental_build.html#sec:task_input_using_classpath_annotations